### PR TITLE
Update local setup

### PR DESCRIPTION
**Pull request by claravanstaden**
Polkadot-sdk companion: https://github.com/Snowfork/polkadot-sdk/pull/110
Resolves: [SNO-826](https://linear.app/snowfork/issue/SNO-826)

- [x] move snowbridge crates to https://github.com/Snowfork/polkadot-sdk
- [x] updated init.sh to checkout the `polkadot-sdk` and symlink it
- [x] e2e stack works
- [x] coverage report works (snowbridge crates' coverage will now be under https://github.com/Snowfork/polkadot-sdk)
- [x] updated benchmark file generation paths

### CODE REVIEW

**Review by vgeddes — APPROVED** [review]
+1

File: scripts/init.sh [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+pushd ..
+if [ ! -d "polkadot-sdk" ]; then
+  git clone https://github.com/Snowfork/polkadot-sdk.git
+fi
+pushd  polkadot-sdk
+git checkout snowbridge
+popd
+popd
+
+ln -sf ../polkadot-sdk polkadot-sdk
```
Comment (alistair-singh) [inline scripts/init.sh:15]: Why do we need this if we are still using the submodule?

**Review by alistair-singh — APPROVED** [review]

**Review by yrong — APPROVED** [review]
+1

File: scripts/init.sh (hunk 1 above)
Comment (yrong) [reply scripts/init.sh:15]: Seems Clara removed it, but we may also need to update git modules accordingly.
https://github.com/Snowfork/snowbridge/blob/c1de2085dfeae6a6b1d155956aaa8bd06b879059/.gitmodules#L1-L3

Comment (claravanstaden) [reply scripts/init.sh:15]: Good catch! 😄 https://github.com/Snowfork/snowbridge/pull/1131/commits/6b4d7457c80850dc0625c555997e28e5879226cc

**Review by alistair-singh — APPROVED** [review]
+1

**Pull request merged**